src/draive/splitters/basic.py

```diff
@@ -33,28 +33,32 @@ def split_text(
 
 def _split(
     text: str,
-    separators: tuple[str, str] | str | None = None,
+    separators: Sequence[str] | str | None = None,
 ) -> tuple[str, list[str]]:
     # prepare initial splitter - default is new paragraph
-    splitter: str
-    alt_splitter: str
+    splitters: Sequence[str]
+
     match separators:
-        case (str(primary), str(secondary)):
-            splitter = primary
-            alt_splitter = secondary
+        case [*separators]:
+            splitters = separators
         case str(primary):
-            splitter = primary
-            alt_splitter = "\n"
+            splitters = [primary]
         case None:
-            splitter = "\n\n"
-            alt_splitter = "\n"
+            splitters = ["\n\n", "\n"]
 
     # try splitting using provided splitter
-    parts: list[str] = text.split(splitter)
-    # if splitting has done nothing retry using secondary splitter
-    if len(parts) == 1:
-        splitter = alt_splitter
+    parts: list[str] = [text]
+    if not splitters:
+        raise ValueError("Missing separators")
+    splitter: str = splitters[0]
```

Comment:
This instruction seems not necessary as we loop through splitters and overwrite this value right after.